### PR TITLE
Fix link

### PR DESCRIPTION
**Pull request by harlantwood**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Same as in Node.js, you just have to [browserify](https://github.com/substack/js
 
 ### In the Browser through `<script>` tag
 
-Make the [ipfsapi.min.js](/ipfsapi.min.js) available through your server and load it using a normal `<script>` tag, this will export the `ipfsAPI` constructor on the `window` object, such that:
+Make the [ipfsapi.min.js](https://github.com/ipfs/js-ipfs-api/blob/master/dist/ipfsapi.min.js) available through your server and load it using a normal `<script>` tag, this will export the `ipfsAPI` constructor on the `window` object, such that:
```
Comment (Dignifiedquire) [inline README.md]: This should be enough: `dist/ipfsapi.min.js`

**Comment by harlantwood** [comment]
@Dignifiedquire since you can view the readme from at least either one of:

- https://github.com/ipfs/js-ipfs-api
- https://github.com/ipfs/js-ipfs-api/blob/master/README.md

I don't think any relative link can work in all cases.

**Comment by diasdavid** [comment]
Thank you @harlantwood :)

**Comment by Mithgol** [comment]
Yep, a relative hyperlink would definitely **not** work from https://www.npmjs.com/package/ipfs-api

**Pull request merged**